### ISSUE TITLE
Posterior humeral circumflex artery occlusion

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11072]
**Preferred term label:**
Posterior humeral circumflex artery occlusion

**Synonyms**
PCHA

**Definition (free text, please give PubMed ID)**
Dye is injected both with patient's humerus at the side of the body and again with shoulder held in abduction and external rotation. Test is positive if there is evidence of PCHA occlusion with arm in abduction and external rotation but artery is patent when arm is held at the side.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Quadrilateral space syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Added as HP:0020318
def: Blockage of the posterior circumflex humeral artery (PCHA), which arises from the third part of the axillary artery and supplies the shoulder joint and a number of its associated muscles. The lesion can be ascertained angiographically, whereby dye is injected both with patient's humerus at the side of the body and again with shoulder held in abduction and external rotation. Test is positive if there is evidence of PCHA occlusion with arm in abduction and external rotation but artery is patent when arm is held at the side.
